static/app/debug/notifications/components/debugNotificationsSidebar.tsx

```diff
@@ -0,0 +1,121 @@
+import {Fragment} from 'react';
+import styled from '@emotion/styled';
+
+import {LinkButton} from 'sentry/components/core/button/linkButton';
+import {Heading} from 'sentry/components/core/text';
+import {notificationCategories} from 'sentry/debug/notifications/data';
+import {useLocation} from 'sentry/utils/useLocation';
+
+export function DebugNotificationsSidebar() {
+  const location = useLocation();
+  return (
+    <CategoryContainer>
+      {notificationCategories.map((category, i) => (
+        <Fragment key={category.value}>
+          {i !== 0 && <CategoryDivider />}
+          <CategoryItem>
+            <CategoryHeading as="h3" size="md">
+              {category.label}
+            </CategoryHeading>
+            <SourceList>
+              {category.sources.map(source => (
+                <SourceItem key={source.value}>
+                  <NotificationLinkButton
+                    borderless
+                    active={location.query.source === source.value}
+                    to={
+                      location.query.source === source.value
+                        ? {query: {...location.query, source: undefined}}
+                        : {query: {...location.query, source: source.value}}
+                    }
+                  >
+                    {source.label}
+                  </NotificationLinkButton>
+                </SourceItem>
+              ))}
+            </SourceList>
+          </CategoryItem>
+        </Fragment>
+      ))}
+    </CategoryContainer>
+  );
+}
+
+const CategoryContainer = styled('div')`
+  display: flex;
+  flex-direction: column;
+  gap: ${p => p.theme.space.xl};
+  padding: ${p => p.theme.space.xl} 0;
+`;
+
+const CategoryDivider = styled('hr')`
+  margin: 0 auto;
+  border-color: ${p => p.theme.tokens.border.muted};
+  width: calc(100% - ${p => p.theme.space.xl});
+`;
+
+const CategoryItem = styled('div')`
+  padding: 0 ${p => p.theme.space.md};
+`;
+
+const CategoryHeading = styled(Heading)`
```

Comment:
If you reorder this, you can remove the styling from Heading. Once [this](https://github.com/getsentry/sentry/pull/96878) lands you could also drop the styled alltogether and set borderBottom="primary"

```tsx
<Container padding="md">
  <Heading>Title</Heading
</Container>
```